src/api/Shrooms.Contracts/Shrooms.Contracts.csproj

```diff
@@ -132,6 +132,7 @@
     <Compile Include="Infrastructure\IExcelBuilder.cs" />
     <Compile Include="Infrastructure\IJobScheduler.cs" />
     <Compile Include="Infrastructure\ILogger.cs" />
+    <Compile Include="Infrastructure\ISortableProperties.cs" />
     <Compile Include="Infrastructure\ISortableProperty.cs" />

```

Comment:
Is this still needed? Seems like you refactored all the places where you added that previously.

Reply:
Removed.